geomesa-index-api/src/main/scala/org/locationtech/geomesa/index/filters/DtgAgeOffFilter.scala

```diff
@@ -100,10 +104,52 @@ object DtgAgeOffFilter {
         i
     }
 
+    // Handle changing dtgIndex for join indexes
+    val updatedDtgIndex = {
+      if (index.name.contains("join")) {
```

Comment:
@elahrvivaz is there a better check for whether or not an index is a join index?  And is that check available in the geomesa-index-api module?